### ISSUE TITLE
Webhook Throws error when try to create certificate

### ISSUE THREAD

**Issue by KamranSwish** [jet/kube-webhook-certgen#17]
Hi 
I am deploying nginx controller in digitalocean kubernetes cluster using terraform.
The nginx ingress controller uses kube-webhook-certgen to create certificate. the pod is created by a job.
the example that I am using is uses "docker.io/jettech/kube-webhook-certgen:v1.2.0" image.

unfortunately in the log of created container I see following error and no secret has been created on the cluster.

> {"error":"invalid configuration: no configuration has been provided","level":"fatal","msg":"error building kubernetes config","source":"k8s/k8s.go:20","time":"2020-07-08T13:43:22Z"} 

I wonder what is the problem?
Thanks

**Comment by vsliouniaev** [comment]
What command-line arguments is the container being started with?

When running in the cluster, the configuration should automatically get loaded from `/var/run/secrets/kubernetes.io/serviceaccount` as described here: https://github.com/kubernetes/client-go/tree/master/examples/in-cluster-client-configuration#authenticating-inside-the-cluster

**Comment by KamranSwish** [comment]
Hi 

Thank you

There was no terraform sample code of yaml files which have used in the project. I converted them to HCL tf files. the default setting for automount_service_account_token  was false. after changing to true previous problem was resolved and everything is running.
Now when I visit http::/ingress-controller-external-ip I see following error in ingress controller deployment log:
Is it related to some SSL certificate Issue?

> 2020/07/10 11:58:28 [error] 36#36: *1964 broken header: "GET /employee HTTP/1.1                                         Host: 157.230.79.145                                                                                                    Connection: keep-alive                                                                                                  DNT: 1                                                                                                                  Upgrade-Insecure-Requests: 1                                                                                            User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36                                                                                                                 Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9                                                                                                            Accept-Encoding: gzip, deflate                                                                                          Accept-Language: sv-SE,sv;q=0.9,fa-IR;q=0.8,fa;q=0.7,en-US;q=0.6,en;q=0.5                                                                                                                                                                       " while reading PROXY protocol, client: 10.114.16.6, server: 0.0.0.0:80                                                 2020/07/10 11:58:28 [error] 36#36: *1965 broken header: "GET /employee HTTP/1.1                                         Host: 157.230.79.145                                                                                                    Connection: keep-alive                                                                                                  Cache-Control: max-age=0                                                                                                Upgrade-Insecure-Requests: 1                                                                                            DNT: 1                                                                                                                  User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36                                                                                                                 Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
Accept-Encoding: gzip, deflate                                                                                          Accept-Language: sv-SE,sv;q=0.9,fa-IR;q=0.8,fa;q=0.7,en-US;q=0.6,en;q=0.5                                                                                                                                                                       " while reading PROXY protocol, client: 10.114.16.6, server: 0.0.0.0:80                                                 2020/07/10 11:58:28 [error] 36#36: *1969 broken header: "GET /employee HTTP/1.1                                         Host: 157.230.79.145                                                                                                    Connection: keep-alive                                                                                                  Cache-Control: max-age=0                                                                                                Upgrade-Insecure-Requests: 1                                                                                            DNT: 1                                                                                                                  User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36                                                                                                                 Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9                                                                                                            Accept-Encoding: gzip, deflate                                                                                          Accept-Language: sv-SE,sv;q=0.9,fa-IR;q=0.8,fa;q=0.7,en-US;q=0.6,en;q=0.5                                                                                                                                                                       " while reading PROXY protocol, client: 10.114.16.6, server: 0.0.0.0:80                                                 2020/07/10 11:58:28 [error] 36#36: *1970 broken header: "GET /employee HTTP/1.1                                         Host: 157.230.79.145                                                                                                    Connection: keep-alive                                                                                                  Cache-Control: max-age=0                                                                                                Upgrade-Insecure-Requests: 1                                                                                            DNT: 1                                                                                                                  User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36                                                                                                                 Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9                                                                                                            Accept-Encoding: gzip, deflate 
Accept-Language: sv-SE,sv;q=0.9,fa-IR;q=0.8,fa;q=0.7,en-US;q=0.6,en;q=0.5                                                                                                                                                                       " while reading PROXY protocol, client: 10.114.16.6, server: 0.0.0.0:80                                                 2020/07/10 11:58:33 [error] 36#36: *1988 broken header: "GET /employee HTTP/1.1                                         Host: 157.230.79.145                                                                                                    Connection: keep-alive                                                                                                  Cache-Control: max-age=0                                                                                                Upgrade-Insecure-Requests: 1                                                                                            DNT: 1                                                                                                                  User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36                                                                                                                 Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9                                                                                                            Accept-Encoding: gzip, deflate                                                                                          Accept-Language: sv-SE,sv;q=0.9,fa-IR;q=0.8,fa;q=0.7,en-US;q=0.6,en;q=0.5                                                                                                                                                                       " while reading PROXY protocol, client: 10.114.16.6, server: 0.0.0.0:80
